host-controller/src/main/java/org/jboss/as/host/controller/IgnoredNonAffectedServerGroupsUtil.java

```diff
@@ -193,21 +190,6 @@ private boolean ignoreServerGroup(final Resource domainResource, final Collectio
         return true;
     }
 
-    private boolean ignoreExtension(final Resource domainResource, final Collection<ServerConfigInfo> serverConfigs, final String name) {
-        //Should these be the subsystems on the master, as we have it at present, or the ones from the slave?
-        Map<String, SubsystemInformation> subsystems = extensionRegistry.getAvailableSubsystems(name);
-        for (String subsystem : subsystems.keySet()) {
-            for (ResourceEntry profileEntry : domainResource.getChildren(PROFILE)) {
-                if (profileEntry.hasChild(PathElement.pathElement(SUBSYSTEM, subsystem))) {
-                    if (!ignoreProfile(domainResource, serverConfigs, profileEntry.getName())) {
-                        return false;
-                    }
-                }
-            }
-        }
-        return true;
-    }
```

Comment:
@boris-unckel Can you comment this out instead, please?

The original use of it at L144 was commented out with the comment '// We don't automatically ignore extensions for now' so this should have been commented at the same time.

I don't recall what that was about but this is complex stuff so I prefer to leave this code around.

Reply:
Thanks for your feedback. I have removed `IgnoredNonAffectedServerGroupsUtil` from the commit. Same here, I'll wait with the `push --force` until further review is done.